pkg/ovs/openflow/utils.go

```diff
@@ -949,6 +966,13 @@ func getFlowModMatch(flowMod *openflow15.FlowMod) string {
 		}
 	}
 
+	// xxregs
```

Comment:
can be removed?

Reply:
Not necessary because the below is used to process OVS xxregs fields.

Reply:
I mean this comment doesn't provide any extra information. Probably remove it or add a clearer comment.

Reply:
removed.